### DISCUSSION TITLE
Using properties for WebSite and WebAppPool names

### DISCUSSION THREAD

**Discussion by Giangi72** [wixtoolset/issues#6310] [Questions] [answered]
Hi all,
I started an issue on WixSharp side but, after some messages, it seems Wix related and not WixSharp.
The original issue is  #https://github.com/oleg-shilo/wixsharp/issues/953#issuecomment-744368685

In few words: I used properties for  WebSite and WebAppPool names, everything works well during install but during unistall the folder is deleted but  WebSite and WebAppPool no.
It seems that, using the properties, the installer loses the IIS items references 

I did 2 tests: one with fixed names and the other using properties, the only differences in wxs files are the guids (obviously) and

```
OK
          <WebSite Id="WebSite" Description="SiteName2" Directory="TestOleg" xmlns="http://schemas.microsoft.com/wix/IIsExtension">
...
         <WebAppPool Id="Server_AppPool" Name="AppPoolName2" ManagedRuntimeVersion="v4.0" ManagedPipelineMode="Integrated" Identity="other" User="newUser" xmlns="http://schemas.microsoft.com/wix/IIsExtension" />

KO
          <WebSite Id="WebSite" Description="[SITENAME]" Directory="TestOleg" xmlns="http://schemas.microsoft.com/wix/IIsExtension">
...
         <WebAppPool Id="Server_AppPool" Name="[APPPOOLNAME]" ManagedRuntimeVersion="v4.0" ManagedPipelineMode="Integrated" Identity="other" User="newUser" xmlns="http://schemas.microsoft.com/wix/IIsExtension" />
```
I also tested fixing web site name and using a variable for application pool = the web site is deleted and application pool is not.
Can anyone help me?
Thanks

My testing environment is
Wix v3.11
Visual Studio Professional 2019 v16.8.3
.Net 4.8
Windows 10 enterprise

**Comment by rseanhall** [comment]
@Giangi72 The WiX Toolset does not use GitHub issues for support. We have converted this issue into a discussion item, which is an experiment that we're trying out. Alternatively, you may want to ask your question on the [wix-users mailing list](https://wixtoolset.org/documentation/mailinglist/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/wix).

**Reply by Giangi72** [reply]
Ok, many thanks

**Comment by wyrdfish** [comment, marked as answer]
The windows installer does not store property information during runs.

https://robmensching.com/blog/posts/2010/5/2/the-wix-toolsets-remember-property-pattern/

**Comment by Giangi72** [comment]
:-)
I was thinking :"while I wait for an answer I can store property values in registry".
I will do it for real now
Many thanks
